Dockerfile

```diff
@@ -1,3 +1,22 @@
+
+###################################
+#Build stage
+FROM golang:1.9-alpine3.6 AS build-env
+
+ARG GITEA_VERSION
+ARG TAGS="bindata sqlite"
+
+#Build deps
+RUN apk --no-cache add git build-base
```

Comment:
I think it is always better to sort packages alphabetically, but this is not a blocker.